src/App.tsx

```diff
@@ -1,9 +1,23 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  KeyboardEvent, useContext, useEffect, useRef, useState,
+} from 'react';
+import { getTodos, addTodo } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Footer } from './components/Footer';
+import { TodosList } from './components/TodosList';
```

Comment:
Create index file for imports

Reply:
create an index file with all imports so that all imports in the app are from one this file